cypress-tests/cypress/e2e/admin/pageBuilder/category/pageCategoriesSearchAndSort.cy.ts

```diff
@@ -0,0 +1,151 @@
+context("Categories Module", () => {
+    const categoryData1 = {
+        name: "ABC",
+        slug: "ABC",
+        layout: "static",
+        url: "/ABC/"
+    };
+    const categoryData2 = {
+        name: "EDF",
+        slug: "EDF",
+        layout: "static",
+        url: "/EDF/"
+    };
+    const categoryData3 = {
+        name: "GHI",
+        slug: "GHI",
+        layout: "static",
+        url: "/GHI/"
+    };
+    const categoryData4 = {
+        name: "!#$%&/()=?*",
+        slug: "!#$%&/()=?*",
+        layout: "static",
+        url: "/!#$%&/()=?*/"
+    };
+    beforeEach(() => {
+        cy.login();
+        cy.pbDeleteAllCategories();
+        cy.wait(1500);
+        cy.createCategory(categoryData1);
+        cy.wait(1500);
+        cy.createCategory(categoryData2);
+        cy.createCategory(categoryData3);
+        cy.wait(1500);
+        cy.createCategory(categoryData4);
+    });
+
+    afterEach(() => {
+        cy.pbDeleteAllCategories();
+    });
+
+    it("Should be able to create, edit, and immediately delete a category", () => {
+        cy.visit("/page-builder/categories");
+        //Assert all of the programatically created categories are being properly displayed.
+        cy.contains(categoryData1.name && categoryData1.url).should("exist");
```

Comment:
```suggestion
        cy.contains(categoryData1.name).should("exist");
        cy.contains(categoryData1.url).should("exist");
```

It's not really about readability, but about how this line of code works:

![image](https://github.com/webiny/webiny-js/assets/5121148/fd2449fe-308e-41c6-b71b-2f6c6f6bd68e)

Basically, this code will always just check for `categoryData1.name` value to exist.